packages/web/pages/onboarding/02.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Box } from '../../components/elements/LayoutPrimitives'
+import { StyledAnchor } from '../../components/elements/StyledText'
+import ExtensionInstallHelp from '../../components/elements/ExtensionsInstallHelp'
+import MobileInstallHelp from '../../components/elements/MobileInstallHelp'
+import Link from 'next/link'
+import OnboardingLayout2 from '../../components/templates/OnboardingLayout2'
+
+const OnboardingPage2 = () => {
+  return (
+    <OnboardingLayout2
+      pageNumber={2}
+      title="Save links to read later"
+      subTitle="Save any link to your library using our apps and browser extensions"
+      description='Install our apps and browser extensions'
+      image={
+        <img
+          src='/static/images/onboarding/browser-extension.svg'
```

Comment:
We had to use an .svg file here as well. Because the .png file would look blurry. Especially the text on it.